examples/react-simple/src/ix/Transaction.js

```diff
@@ -1,37 +1,162 @@
+      - The payer specifies the Account for which will pay for the transaction.
+      - Each authorizer specifies an AuthAccount which can be used within the cadence code of the transaction.
+
+      To declare a proposer, authorizer or payer, you must specify an authorization.
+      The sdk.authorization constructor consumes an account address, a signing function, and a keyId.
+
+      The signing function is a function with an interface as such:
+
+          const signingFunction = ({
+            message,          // The message to be signed.
+            addr,             // The address specified in the authorization.
+            keyId,            // The keyId specified in the authorization.
+            roles: {      
+              proposer,       // Denotes if this authorization is acting as a proposer.
+              authorizer,     // Denotes if this authorization is acting as a authorizer.
+              payer,          // Denotes if this authorization is acting as a payer.
+            },
+            interaction,      // The full interaction object such that the message can be reconstructed if desired, for security.
+          }) => { ... }
+
+      The signing function must return an object as such:
+
+          return {
+            addr,       // The address of the Flow Account which produced this signature.
+            keyId,      // The keyId used to produce the signature.
+            signature   // A hex encoded string of the signature produced by this function.
+          }
+      
+      The signing function produces a signature of the message using the key with the keyId as specified.
+
+      Arguments
+      ---------
+
+      Transactions can also contain arguments. Arguments are variables that are passed into a transaction.
+      Specifying arguments is done by specifying an array of arguments.
+      Each argument consumes a JavaScript value, and an associated Cadence type identifier.
+      
+      Denoting an argument is done by calling the args and arg builder as such:
+
+          sdk.args([ sdk.arg("my string value", t.String) ])
+
+      Validators
+      ----------
+
+      To check that everything has gone as expected while building and resolving your transaction, validators act as a
+      mechanism to check the interaction object has been formulated to your desired specification.
+
+      Lets assume you expect only one argument to be included in this transaction interaction, a validator for such looks like:
+      
+          sdk.validator((ix, {Ok, Bad}) => {
+            if (Object.keys(ix.arguments).length > 1) return Bad(ix, "This transaction should only have one authorization!")
+            return Ok(ix)
+          })
+
+      Resolvers
+      ---------
+
+      Before a transaction interaction is ready to be sent, it must, if not yet ready, be first be passed through a sequence of resolvers.
+      Resolvers fill in and further prepare an interaction into a state where it is ready to be sent.
```

Comment:
```suggestion
      Resolvers gather the information necessary to finish building the finalized interaction, so it's in a state which is valid and ready to be sent to Flow.
```

Reply:
Something like this. Introducing the term "interaction" here might throw some people off.

Reply:
i think its ok to talk about interactions here in the js-sdk example since this is meant for folks concerned about the low-level implementation.